Documentation tests currently failing on main

When using doctest, this section of documented code in main (`doc/support/arrays.rst`), this error now occurs, causing a cascade of failing CI tests on PRs that have been brought up-to-date with main.  Unsure why this has stopped working.  

```
=================================== FAILURES ===================================
_____________________________ [doctest] arrays.rst _____________________________
061    >>> nodes = thaw(dcoll.nodes(), actx)
062    >>> dd = DOFDesc("all_faces")
063    >>> nhat = thaw(dcoll.normal(dd), actx)
064 
065 What can go wrong?  Attempts to operate on frozen data will yield errors similar to
066 the following:
067 
068 .. doctest::
069 
070    >>> dcoll.nodes() * 5
UNEXPECTED EXCEPTION: ValueError('PyOpenCL array has no queue; call .with_queue() to add one in order to be able to perform operations')
Traceback (most recent call last):
  File "/home/runner/work/mirgecom/mirgecom/.miniforge3/envs/testing/lib/python3.8/doctest.py", line 1336, in __run
    exec(compile(example.source, filename, "single",
  File "<doctest arrays.rst[13]>", line 1, in <module>
  File "<container arithmetic for DOFArray>", line 123, in _dofarray_mul
  File "<container arithmetic for DOFArray>", line 123, in <listcomp>
  File "/home/runner/work/mirgecom/mirgecom/.miniforge3/envs/testing/lib/python3.8/site-packages/pyopencl/array.py", line 1158, in __mul__
    common_dtype = _get_common_dtype(self, other, self.queue)
  File "/home/runner/work/mirgecom/mirgecom/.miniforge3/envs/testing/lib/python3.8/site-packages/pyopencl/array.py", line 55, in _get_common_dtype
    raise ValueError("PyOpenCL array has no queue; call .with_queue() to "
ValueError: PyOpenCL array has no queue; call .with_queue() to add one in order to be able to perform operations
/home/runner/work/mirgecom/mirgecom/doc/support/arrays.rst:70: UnexpectedException
```

--- a/doc/support/arrays.rst
+++ b/doc/support/arrays.rst
@@ -70,10 +70,8 @@ the following:
    >>> dcoll.nodes() * 5
    Traceback (most recent call last):
     ...
-   AttributeError: 'NoneType' object has no attribute 'device'
+   ValueError: PyOpenCL array has no queue; call .with_queue() to add one in order to be able to perform operations
 
-(That error message is not ideal; it is being `worked on
-<https://github.com/inducer/pyopencl/pull/486>`__.)
 Fortunately, recovering from this is straightforward:
 
 .. doctest::

